src/plugins/data/public/ui/query_string_input/_query_bar.scss

```diff
@@ -19,7 +19,7 @@
 }
 
 .kbnQueryBar__textarea {
-  @include kbnThemeStyle('v7') {
+  @include kbnThemeStyle('vs7') {
```

Comment:
I might be lost again :) 
What does `vs7` do in this case?

Reply:
Whoops, sorry I was testing the validation. It's fixed now.